Arelle/Arelle
# Detect and Load TkTable Library

#### Reason for change
Relates to #1347. Note there are still exceptions raised in rendering the table linkbase caused by #1111, but this allows Arelle to automatically discover and load the tktable library that's packaged in the repo.

#### Description of change
* Add TkTable libs for macos arm64.
* Detect and load TkTable libs when running from both source and frozen builds.

#### Steps to Test
* Verify that "Unable to locate or load Tktable" error is not raised when opening a taxonomy that has a table linkbase in the GUI with both the frozen builds and from source on Windows, macOS, and Linux.

**review**:
@Arelle/arelle
